build broken with gcc 10.2.0

Everything works fine with gcc 9.3.0 (Ubuntu 20.04).
But fails with gcc 10.2.0 (Ubuntu 20.10)
```
[ 98%] Linking C executable ujit-parse-profile
/usr/bin/ld: CMakeFiles/ujit-parse-profile.dir/parse_profile/ujpp_table.c.o:/home/user/checkout/luavela/tools/parse_profile/ujpp_demangle.h:15: multiple definition of `v'; CMakeFiles/ujit-parse-profile.dir/parse_profile/ujpp_main.c.o:/home/user/checkout/luavela/tools/parse_profile/ujpp_demangle.h:15: first defined here
/usr/bin/ld: CMakeFiles/ujit-parse-profile.dir/parse_profile/ujpp_utils.c.o:/home/user/checkout/luavela/tools/parse_profile/ujpp_demangle.h:15: multiple definition of `v'; CMakeFiles/ujit-parse-profile.dir/parse_profile/ujpp_main.c.o:/home/user/checkout/luavela/tools/parse_profile/ujpp_demangle.h:15: first defined here
/usr/bin/ld: CMakeFiles/ujit-parse-profile.dir/parse_profile/ujpp_parser.c.o:/home/user/checkout/luavela/tools/parse_profile/ujpp_demangle.h:15: multiple definition of `v'; CMakeFiles/ujit-parse-profile.dir/parse_profile/ujpp_main.c.o:/home/user/checkout/luavela/tools/parse_profile/ujpp_demangle.h:15: first defined here
/usr/bin/ld: CMakeFiles/ujit-parse-profile.dir/parse_profile/ujpp_callgraph.c.o:/home/user/checkout/luavela/tools/parse_profile/ujpp_demangle.h:15: multiple definition of `v'; CMakeFiles/ujit-parse-profile.dir/parse_profile/ujpp_main.c.o:/home/user/checkout/luavela/tools/parse_profile/ujpp_demangle.h:15: first defined here
/usr/bin/ld: CMakeFiles/ujit-parse-profile.dir/parse_profile/ujpp_demangle.c.o:/home/user/checkout/luavela/tools/parse_profile/ujpp_demangle.h:15: multiple definition of `v'; CMakeFiles/ujit-parse-profile.dir/parse_profile/ujpp_main.c.o:/home/user/checkout/luavela/tools/parse_profile/ujpp_demangle.h:15: first defined here
/usr/bin/ld: CMakeFiles/ujit-parse-profile.dir/parse_profile/ujpp_output.c.o:/home/user/checkout/luavela/tools/parse_profile/ujpp_demangle.h:15: multiple definition of `v'; CMakeFiles/ujit-parse-profile.dir/parse_profile/ujpp_main.c.o:/home/user/checkout/luavela/tools/parse_profile/ujpp_demangle.h:15: first defined here
/usr/bin/ld: CMakeFiles/ujit-parse-profile.dir/parse_profile/ujpp_read.c.o:/home/user/checkout/luavela/tools/parse_profile/ujpp_demangle.h:15: multiple definition of `v'; CMakeFiles/ujit-parse-profile.dir/parse_profile/ujpp_main.c.o:/home/user/checkout/luavela/tools/parse_profile/ujpp_demangle.h:15: first defined here
/usr/bin/ld: CMakeFiles/ujit-parse-profile.dir/parse_profile/ujpp_state.c.o:/home/user/checkout/luavela/tools/parse_profile/ujpp_demangle.h:15: multiple definition of `v'; CMakeFiles/ujit-parse-profile.dir/parse_profile/ujpp_main.c.o:/home/user/checkout/luavela/tools/parse_profile/ujpp_demangle.h:15: first defined here
/usr/bin/ld: CMakeFiles/ujit-parse-profile.dir/parse_profile/ujpp_demangle_lua.c.o:/home/user/checkout/luavela/tools/parse_profile/ujpp_demangle.h:15: multiple definition of `v'; CMakeFiles/ujit-parse-profile.dir/parse_profile/ujpp_main.c.o:/home/user/checkout/luavela/tools/parse_profile/ujpp_demangle.h:15: first defined here
/usr/bin/ld: CMakeFiles/ujit-parse-profile.dir/parse_profile/ujpp_elf.c.o:/home/user/checkout/luavela/tools/parse_profile/ujpp_demangle.h:15: multiple definition of `v'; CMakeFiles/ujit-parse-profile.dir/parse_profile/ujpp_main.c.o:/home/user/checkout/luavela/tools/parse_profile/ujpp_demangle.h:15: first defined here
collect2: error: ld returned 1 exit status
```

--- a/tools/parse_profile/ujpp_demangle.h
+++ b/tools/parse_profile/ujpp_demangle.h
@@ -12,7 +12,6 @@
 #include "ujpp_vector.h"
 #include "ujpp_utils.h"
 
-struct vector v;
 struct parser_state;
 struct hvmstate;
 

